tests/reporters/test_custom_flex_reporter.py

```diff
@@ -11,7 +11,7 @@
 from telliot_feeds.datafeed import DataFeed
 from telliot_feeds.feeds.eth_usd_feed import eth_usd_median_feed
 from telliot_feeds.feeds.matic_usd_feed import matic_usd_median_feed
-from telliot_feeds.reporters.custom_flex_reporter import CustomFlexReporter
+from telliot_feeds.reporters.custom_flex_reporter import Custom360Reporter
```

Comment:
Thx for spotting this. I forgot to remove that import. I basically removed the custom reporter classes, since there was only a few lines of code difference between those classes and what they were inheriting from (360/interval reporter). It seems like you can just swap the addresses if they want to use a diff contract to report. Though a function `verify_custom_contract_abi` might need to be added. Currently it assumes their custom contract ABI will work with the one they're overwriting (oracle/autopay/token).